Assets/Scripts/Destructable.cs

```diff
@@ -10,7 +10,6 @@ void OnCollisionEnter(Collision collisionInfo)
     {
         if (collisionInfo.collider.tag == "Player")
         {
-            Debug.Log("Test");
             Instantiate(destroyedVersion, transform.position, transform.rotation);
             Destroy(gameObject);
         }
```

Comment:
A comment below this says `//todo-ck add a timer to 'destroy' the pieces when they're offscreen`

You probably already knew this but a possibly better alternative is to just make your `DetectionZone` 'garbage collect' these already destroyed destructables

Reply:
I didn't think of that actually but it makes total sense! Thanks!